ods_ci/tests/Tests/1300__model_registry/1302_model_registry_model_serving.robot

```diff
@@ -248,3 +250,26 @@ Run Update Notebook Script
     Log    ${result.stdout}
     Log    ${result.stderr}
     Should Contain    ${result.stdout}    Modified notebook saved
+
+Remove Model Registry
+    [Documentation]    Run multiple oc delete commands to remove model registry components
+    Run OC Delete Command    oc delete smm default -n ${NAMESPACE_MODEL-REGISTRY}
+    Run OC Delete Command    oc delete -k ${MODELREGISTRY_BASE_FOLDER}/samples/secure-db/mysql-tls
+    Run OC Delete Command    oc delete secret modelregistry-sample-grpc-credential -n ${NAMESPACE_ISTIO}
+    Run OC Delete Command    oc delete secret modelregistry-sample-rest-credential -n ${NAMESPACE_ISTIO}
+    Sleep    30s
```

Comment:
if you need to wait because of previous deletion may still be ongoing, you could use the oc wait for deletion command 
`oc wait --for=delete <resource> <name> -n <ns>`